Capstone project

hetrox8 opened the pull request:
This is an online institution website .where you can access a variety of courses. The main languages used to accomplish this page include html , css and javascript.

here is a link for the demo https://www.loom.com/share/053a36b874794ebfb42e34a7611b87ba?sid=8b47dd33-225e-4ca2-9fa5-7b4997cc3733

hetrox8 committed:
chnge

hetrox8 committed:
make changes

hetrox8 committed:
make changes

hetrox8 committed:
make changes

hetrox8 committed:
make changes

hetrox8 committed:
make

hetrox8 committed:
make changes

hetrox8 committed:
make

hetrox8 committed:
make

hetrox8 committed:
make

hetrox8 committed:
make changes

hetrox8 committed:
make

hetrox8 committed:
make changes

hetrox8 committed:
fix linters

hetrox8 committed:
fix  eslint

hetrox8 committed:
change layout

hetrox8 committed:
fix slinters

hetrox8 committed:
changes

hetrox8 committed:
fix liners

Cabraham1 commented on about.html:
```diff
@@ -29,37 +29,20 @@
                 </ul>
-                <li><a href=""> <img class="logo" src="./images/logo.png" alt=""></a> </li>  
             </div>
       
```
- [ ] Nice work so far 👍🏿  as per project requirements a user should be able to navigate between the home and about us pages but I notice that the about us page does not have a link to go back to the home page. I kindly suggest you add navigation to allow users to go back to the home page on the about us  page

Cabraham1 commented on about.html:
```diff
@@ -29,37 +29,20 @@
+            <img src="./images/mobile_menu 1.png" alt="">
+        </div>
+       
+<div class="frame-1">
```
- [ ] As per project requirement i kindly suggest you make your home and about us page have the same navbar and header background

Cabraham1 commented on styles.css:
```diff
@@ -46,31 +59,49 @@ nav {
 }
```
- [ ] As per project requirement you are to use the same background image for your project. Kindly replace your background image with the design image

Cabraham1 commented on index.html:
```diff
@@ -21,20 +21,33 @@
+            
       
     </header>
     <section class="container contain">
+        
         <div class="mobi-logo">
-            <img class="logo-1" src="./images/mobile_menu 1.png" alt="">
+            <img id="hammbuger" class="logo-1" src="./images/mobile_menu 1.png" alt="">
+            <img class="x-icon" id="x-icon" src="./images/Icon - Cancel.svg" alt="">
+            <img class="lop" src="./images/logo.png" alt="">
```
- [ ] Nice work so far for accessibility kindly add all the image att attributes your entire application

Cabraham1 commented on index.html:
```diff
@@ -10,7 +10,7 @@
     <header>   
             <nav> 
-                <img class="logo" src="" alt="">
+                <img class="un-k" src="" alt="">
                    <ul class="first-li">
                        <li><a href=""><img src="./images/facebook.png" alt=""></a></li>
                        <li><a href=""><img src="./images/twitter.png" alt=""></a></li> 
```
- [ ] kindly add a hyperlink to your logo  that will redirect to home and also add the image attributes

Cabraham1 reviewed (changes requested):
# Required Changes :recycle:
![almost-there](https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif)

## Hey @hetrox8  :wave:

## Highlights :1st_place_medal:

✔️ Impressive UI
✔️ Good use of Javascript arrays
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
- [ ] I kindly suggest you adjust your media query as your application is not responsive from 786 upward
- [ ] I kindly request that you adhere to the Figma design provided for your project. The Figma design serves as a visual reference and guideline for implementing the desired user interface and overall look of your application. Please ensure that the implemented code aligns with the design specifications provided in Figma. This includes aspects such as layout, color scheme, typography, and any other visual elements defined in the design.

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hetrox8 committed:
fix linters

hetrox8 commented:
1.implemented  feature for user to navigate from home to about page
2.changed home and about page use the same nav
3.Replaced the background image as per the design
4.Added 'alt ' text for accessibility

ishimwezachee commented on about.html:
```diff
@@ -29,37 +29,20 @@
                     <li><a href="">sponsor</a></li
                     <li><a href="#">News</a></li>
```
- [ ]  Previous reviewer suggested you to have the same navigation in your project as it from the provided design. Right now About page navigations are in left sides while the home navigations are in the right sides. Just make sure that you have uniform navigations on both pages.

ishimwezachee commented on about.html:
```diff
@@ -29,37 +29,20 @@
+<div class="frame-1">
     <section class="about-page" id="about-page">
         <h1 class="heading-1">"Hetroxity Academy"</h1>
+        <div class="yy">
+            life changing platform . discover your full potential
+        </div>
        <div class="abt-p">
```
- [ OPTIONAL] Previous reviewer did not mention but this part is of about me is totally different from the one in the design. Please fix it as well so that the image and footer / partner's are both position as well as the provided design. 

![image](https://github.com/hetrox8/First-Capstone-Project/assets/43877689/e34e7cba-f8d8-4e06-8241-dda93dd568cf)

ishimwezachee reviewed (changes requested):
Hi  @hetrox8 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![image](https://media.tenor.com/zr9E2WI4EvMAAAAM/almost-there-reach.gif)

 ## Highlights
✔️ Good UI
✔️ Implemented some of the changes from previous reviewer 
✔️ Following some requirements

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hetrox8 committed:
fix  nav layout

hetrox8 commented:
1. fixed nav bar to display correctly  as in home and about page
2. [optional change] fixed partner section design flow.

skyv26 commented on about.html:
```diff
@@ -5,61 +5,44 @@
+            </ul>
+        </nav>
+
+      <div class="second-item">
+        <img class="logo" src="./images/logo.png" alt="logo icon"> 
+             <ul class="second-li">
+                <li><a class="active" href="index.html">home</a></li>
+                <li><a href="about.html">about</a></li>
+                <li><a href="">join</a></li>
+                <li><a href="">sponsor</a></li
+                <li><a href="#">News</a></li>
+            </ul>
+           
+        </div>
```
- [ ] Well done 👏🏻 , you did an awesome job by implementing the changes as per the previous code reviewer's request 👍🏻 but still your `about.html` navigation is not looking similar to your `home.html` page. Kindly fix it because right now your design does not look professional at all. I would recommend you also fix other things in your project to make it looks like the original design.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Home Page Design</th>
<th>About Page Design</th>
</tr>
<tr>
<td>

![image](https://github.com/hetrox8/First-Capstone-Project/assets/66476329/547b94b9-8b88-46a3-ba8e-a0ad2a8e3c1d)

</td>
<td>

![image](https://github.com/hetrox8/First-Capstone-Project/assets/66476329/95f49c56-d148-4de6-8130-1f43be019df0)

</td>
</tr>
</table>

hetrox8 replied:
omg i just can't believe this i literally implement everything each code reviewer ask for. i am having limited chances and this is my last chance else i will be withdrwan from the program.

skyv26 reviewed (changes requested):
## Hi @hetrox8, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## STATUS: REQUIRE CHANGES 🔴 

![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExYTY5MTMxNDUzNWFmNjU5MmFjOWZjY2IyMDIyY2RhNWFlYmU4NjdlNCZlcD12MV9pbnRlcm5hbF9naWZzX2dpZklkJmN0PWc/Xfb7kQ0VhUIrpiuGx1/giphy.gif)

## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   <img src="https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white" alt="@skyv26" />
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   <img src="https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white" alt="linkedin-skyv2022" />
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   <img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white" alt="@vrma_aakash" />
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   <img src="https://img.shields.io/badge/Instagram-E4405F?style=for-the-badge&logo=instagram&logoColor=white" alt="@skybrel" />
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   <img src="https://user-images.githubusercontent.com/66476329/243158690-61eb1021-fbb6-464d-b841-a09c4caa156e.png" alt="@skybrel" />
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hetrox8 committed:
make changes

hetrox8 committed:
fix nav

hetrox8 commented:
.1.fixed nav bar

Shisui6 reviewed (approved):
Hi @hetrox8, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
